cargo/flash.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+set -e
+
+BUILD_MODE=""
+case "$1" in
+    ""|"release")
+        bash build.sh
+        BUILD_MODE="release"
+        ;;
+    "debug")
+        bash build.sh debug
+        BUILD_MODE="debug"
+        ;;
+    *)
+        echo "Wrong argument. Only \"debug\"/\"release\" arguments are supported"
+        exit 1;;
+esac
+
+export ESP_ARCH=
+{%- if mcu == "esp32c3" -%}
+riscv32imc-esp-espidf
+{%- else -%}
+xtensa-{{ mcu }}-espidf
+{%- endif %}
+
+web-flash --chip {{ mcu }} target/${ESP_ARCH}/${BUILD_MODE}/{{ crate_name }}
```

Comment:
The Cargo.toml uses `name = "{{project-name}}"`. If a project name contains `-`, `{{crate_name}}` replaces that with a `_` and therefore flash.sh and run-wokwi.sh will not find the binary file.